docs/conventions.md

```diff
@@ -49,9 +49,9 @@ The `kyt test` command finds all files with `*.test.js` in the `/src` directory.
 
 The `kyt lint` command finds all files with `.js` extension in `/src`
 
-Specify lint rules with a `.eslintrc` file in the root of your project.
+Specify lint rules with a `.eslintrc.json` file in the root of your project.
```

Comment:
I updated the docs. In this section I mention and link to `setup` and then in the setup command documentation, I mentioned that we backup.